examples/cpp/basic.cpp

```diff
@@ -22,4 +22,5 @@ int main() {
   opendal::ReaderStream stream(reader);
   std::string res2;
   stream >> res2; // res2 == "abc"
+  std::cout<<res2<<std::endl;
```

Comment:
We have a comment on the last line to show the result. 
`// res2 == "abc"`
I don't think it need to add a new line to display it to standard output.

Reply:
yes, I noticed the comment. But for a newbie，when he run a program, there is no any outputs and the program terminated. This seems a little strange. So I think print the result is better.